src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -925,7 +925,12 @@ class DevTask extends AbstractFeatureTask {
 
         if (serverStartTimeout == null) {
             if (server.timeout != null && server.timeout.isInteger()) {
-                serverStartTimeout = server.timeout.toInteger();
+                try {
```

Comment:
I don't believe `isInteger()` is valid on the line before this either. Probably should just check that the string is notEmpty().

Reply:
Yeah, missed that. Fixed it.